src/Planner/grasp_planner.cpp

```diff
@@ -585,13 +585,13 @@ grasp_planner::get_planningParameters(int& nr_of_360_deg_steps_in,
 SoPathList 
 grasp_planner::searchPrimitives(GraspableBody* bg)
 {	
+  Q_UNUSED(bg)
```

Comment:
Perhaps remove argument name from function declaration and definition: grasp_planner::searchPrimitives(GraspableBody*)